backend/server.py

```
@@ -1501,16 +1501,27 @@ async def lifespan(app_: "FastAPI"):
+    # /api/auth/login is not delayed by schedule deduplication. The cleanup
+    # is idempotent + best-effort — the per-minute Cloudflare Cron
+    # /api/scheduler/tick also runs force_cleanup on every tick, so cleanup
+    # still happens within 60s of cold start. Don't block the first request
+    # on cleanup that can happen 60s later with no user-visible impact.
+    async def _deferred_startup_cleanup() -> None:
+        try:
+            cleanup = await SCHEDULER.force_cleanup()
+            if cleanup.get("deleted", 0) > 0 or cleanup.get("deduped", 0) > 0:
+                log.info("Startup schedule cleanup: deleted=%d deduped=%d total=%d",
+                         cleanup["deleted"], cleanup["deduped"], cleanup["total"])
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)